javav2/example_code/emr/src/main/java/aws/example/emr/DescribeCluster.java

```diff
@@ -0,0 +1,71 @@
+//snippet-sourcedescription:[CreateSparkCluster.java demonstrates how to create and start running a new cluster (job flow).]

```

Comment:
Wrong description/filename for this file.

Reply:
fixed